app/helpers/form_helper.rb

```diff
@@ -20,6 +20,14 @@ def block_date_field_tag(name, value, options = {})
       form_input_error_messages(name, options)
   end
 
+  def idv_using_mock_vendor
+    if Figaro.env.proofing_vendors
+      Figaro.env.proofing_vendors.split(/\W+/).map(&:to_sym).include?(:mock)
+    else
+      true
```

Comment:
Its the rails way to do "return true unless Figaro.env.proofing_vendors"? ..  not that I like this pattern...